label_studio/tasks/migrations/0001_squashed_0041_taskcompletionhistory_was_cancelled.py

```diff
@@ -64,7 +64,7 @@ class Migration(migrations.Migration):
                 ('created_at', models.DateTimeField(auto_now_add=True, help_text='Creation time', verbose_name='created at')),
                 ('updated_at', models.DateTimeField(auto_now=True, help_text='Last update time', verbose_name='updated at')),
                 ('completed_by', models.ForeignKey(help_text='User who made this completion', null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='completions', to=settings.AUTH_USER_MODEL)),
-                ('task', models.ForeignKey(help_text='Corresponding task for this completion', null=True, on_delete=django.db.models.deletion.CASCADE, related_name='completions', to='tasks.task')),
+                ('task', models.ForeignKey(help_text='Corresponding task for this completion', null=True, on_delete=django.db.models.deletion.CASCADE, related_name='completions', to='tasks.task', db_index=False)),
```

Comment:
Why did you use db_index=False?

Reply:
If we don't set it, the migrations fail with:

```
Cannot drop index 'task_comple_task_id_07c6ca_idx': needed in a foreign key constraint
```
A database index is automatically created when we use `models.ForeignKey` which prevents the following migration on MySQL:
```
migrations.RemoveIndex(
    model_name='annotation',
    name='task_comple_task_id_07c6ca_idx',
),
```
Maybe this `db_index=False` can be computed if `DJANGO_DB` is `mysql`

Reply:
It would be great if you could rewrite it using DJANGO_DB.

Reply:
Hey @makseq, I have added a change to check if `DJANGO_DB` is `mysql`.